### PR TITLE
How-tos updated outputs to match all except python

### PR DESCRIPTION
**Pull request by anistark**
# Description of change

How-tos Alias outputs match. Except python.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Fix

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: sdk/examples/how_tos/alias/create.rs [hunk 1]
```diff
@@ -44,7 +44,7 @@ async fn main() -> Result<()> {
         .retry_transaction_until_included(&transaction.transaction_id, None, None)
         .await?;
     println!(
-        "Block included: {}/block/{}",
+        "Transaction included: {}/block/{}",
```
Comment (Thoralf-M) [inline sdk/examples/how_tos/alias/create.rs]: This should stay Block (we have that 57 times, `Transaction included` only 3 times)

Comment (anistark) [reply sdk/examples/how_tos/alias/create.rs]: Ok will replace the rest to `Transaction` then.

Comment (Thoralf-M) [reply sdk/examples/how_tos/alias/create.rs]: But the link has the block and not transaction

**Review by DaughterOfMars — APPROVED** [review]

**Review by kwek20 — DISMISSED** [review]

File: bindings/python/examples/how_tos/alias/create.py [hunk 2]
```diff
@@ -20,4 +20,4 @@
 
 # Send transaction.
 transaction = account.prepare_create_alias_output(None, None).send()
-print(f'Block sent: {os.environ["EXPLORER_URL"]}/block/{transaction.blockId}')
+print(f'Transaction sent: {os.environ["EXPLORER_URL"]}/block/{transaction.blockId}')
```
Comment (thibault-martinez) [inline bindings/python/examples/how_tos/alias/create.py]: This should stay Block ?

Comment (kwek20) [reply bindings/python/examples/how_tos/alias/create.py:23]: Then we should change it in rust instead

Comment (anistark) [reply bindings/python/examples/how_tos/alias/create.py]: I believe it's "Transaction Sent" and "Block included" everywhere.
Matching it with rust example.

Comment (Thoralf-M) [reply bindings/python/examples/how_tos/alias/create.py]: But then it should be also the transaction url, block inclusion is not checked for in this example

Comment (thibault-martinez) [reply bindings/python/examples/how_tos/alias/create.py]: @anistark ?

Comment (thibault-martinez) [reply bindings/python/examples/how_tos/alias/create.py]: Let's try to merge this ASAP pls

**Review by DaughterOfMars — APPROVED** [review]

File: bindings/python/examples/how_tos/alias/create.py (hunk 2 above)
Comment (Thoralf-M) [inline bindings/python/examples/how_tos/alias/create.py]:
```suggestion
print(Block sent: {os.environ["EXPLORER_URL"]}/block/{transaction.blockId}')
```

**Review by DaughterOfMars — DISMISSED** [review]

File: bindings/python/examples/how_tos/alias/create.py [hunk 3]
```diff
@@ -20,4 +20,4 @@
 
 # Send transaction.
 transaction = account.prepare_create_alias_output(None, None).send()
-print(f'Block sent: {os.environ["EXPLORER_URL"]}/block/{transaction.blockId}')
+print(Block sent: {os.environ["EXPLORER_URL"]}/block/{transaction.blockId}')
```
Comment (DaughterOfMars) [inline bindings/python/examples/how_tos/alias/create.py]: goof?

Comment (Thoralf-M) [inline bindings/python/examples/how_tos/alias/create.py]:
```suggestion
print(f'Block sent: {os.environ["EXPLORER_URL"]}/block/{transaction.blockId}')
```

**Review by Thoralf-M — DISMISSED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**